app/components/TasksToolbar.tsx/ColorSelector.tsx

```diff
@@ -7,13 +7,16 @@ import { COLORS } from '@/utils/constants';
 
 const ColorSelector = ({
```

Comment:
I've moved away from assigning colors to tasks separately from tags - I like the more straightforward route of "tasks are colored according to the tag they fall under" as all tags can be assigned a color. So this component, which was originally a color selector inside the AddTasks input is now more of a "TagPreview" component, and we should rename it as such.